### PR TITLE
[new] Added event-drops w/ npm auto-update via single package.json 

### PR DESCRIPTION
**Pull request by austin665**
Pull request for issue: #11999 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/marmelab/EventDrops
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 40
   - Count of stars: 792
   - Count of forks: 157
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/event-drops/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "svg",
+    "animation",
+    "canvas"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marmelab/EventDrops"
+  },
+  "license": "MIT",
+  "autoupdate": {
```
Comment (maruilian11) [inline ajax/libs/event-drops/package.json]: Hi @austin665, I think the files we need are in `dist` in npm package. Could you remove the `git auto-update` and change to use:
```
"npmFileMap": [
  {
    "basePath": "dist",
    "files": [
      "**/*"
    ]
  }
]
```

File: ajax/libs/event-drops/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "event-drops",
+  "filename": "index.min.js",
```
Comment (maruilian11) [inline ajax/libs/event-drops/package.json]: Please change to `"eventDrops.js"` instead.

Comment (austin665) [reply ajax/libs/event-drops/package.json]: Hi @maruilian11 thanks for the review. Should it be eventDrops.min.js or eventDrops.js ?

**Comment by maruilian11** [comment]
Hi @austin665, sorry for not noticing that there is already a PR working on adding this lib. I'm afraid I need to close this one because it opened later. Feel free to pick another issue and thank you for your contribution.
https://github.com/cdnjs/cdnjs/pull/12000

**Pull request closed without merging**